docs setup/Readme.md

```diff
@@ -0,0 +1,67 @@
+# Guide to Run Documentation Locally
+
+## 1. Install Poetry
+
+Run the following command in your project directory:
+
+```bash
+pip install poetry
+```
+
+This will install [Poetry](https://python-poetry.org/).
+
+---
+
+## 2. Install Dependencies
+
+* **On macOS/Linux:**
+
+```bash
+poetry install
+```
+
+* **On Windows (Git Bash / PowerShell):**
+
+```bash
+poetry install
+```
+
+---
+
+## 3. Activate the Virtual Environment
+
+Find the environment path and activate it:
+
+```bash
+poetry env activate
+```
+
+This will print the full path of the environment. Copy it and source it manually, for example:
+
+```bash
+source /path/to/venv/bin/activate
+```
+
+Once activated, you’re inside the Poetry-managed virtual environment.
```

Comment:
You need to activate the environment first. Otherwise you're installing dependencies on your current environment, rather than the virtual environment target.